dvc/stage/__init__.py

```diff
@@ -472,7 +475,7 @@ def run(
             self.remove_outs(ignore_remove=False, force=False)
 
         if not self.frozen and self.is_import:
-            sync_import(self, dry, force)
+            sync_import(self, dry, force, jobs=self.jobs)
```

Comment:
I think you should be able to avoid setting and using `self.jobs` and `PARAM_JOBS`. Just pass `jobs=...` to `run` in `dvc/repo/imp_url.py`

Reply:
I had considered this before but `stage.run` didn't have any variable before. This represents that it all depends on member variables not on the running status. Adding a `variable` to it break this, and this would cause some inconsistency of `dvc repo`. 

> Feels like there should be a config-level thing to set it. There is a ticket for it already, IIRC. So I suggest just leaving as is for now. Very good observation though!

Maybe we should abandon `--jobs` and makes config effect?

Reply:
@karajan1001 Good point! Important consistency here is fetch/pull and import. `repro` is a bit odd like that with `import`s, where it tries to `run` instead of trying to `pull`. That's something that we are trying to fix with `dvc repro --pull` flag right now that will become default behaviour in the future. So ideally there should indeed be a way of telling `import` that it should load config options from some config section and it should indeed save it in dvcfile. Smth like
```
remote:
  config:
    from-remote: myremote  # handy to not have to remember remote name in the project you are importing from
```
if we can make a generalization like that, then `--jobs` would indeed be no longer needed. This would be a more holistic solution, no doubt about that.